.github/workflows/quality.yml

```diff
@@ -12,43 +12,43 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
```

Comment:
CI checks didn't work because you modified the CI file `quality.yml`. In the future, do **NOT** modify files under `.github/workflows/` without prior discussion.

Since your proposed formatting is indeed the good one, I applied the change directly into `dev`. Please rebase. The CI checks should execute after your rebase.